drivers/sensor/st/lsm6dsv16x/lsm6dsv16x_trigger.c

```diff
@@ -133,6 +133,10 @@ int lsm6dsv16x_trigger_set(const struct device *dev,
 		return -EINVAL;
 	}
 
+	if (!lsm6dsv16x_is_active(dev)) {
+		return -EBUSY;
+	}
+
```

Comment:
OK, but you should also protect lsm6dsv16x_submit() I guess.
You have to do all the driver's API.